server/index.js

```diff
@@ -1,76 +1,79 @@
-
-app.listen(process.env.SERVER_PORT, (err) => {
-  if (err) {
-    console.log(err);
-  } else {
-    console.log(`Server is running on ${process.env.SERVER_PORT}`);
-  }
-});
-
-app.use(bodyParser.json());
-
-app.use('/', express.static(path.join(__dirname, '../client/dist/')));
-
-
-app.get('/popularDishes/getCompany', (req, res) => {
-  model.getCompany(companyNumber, (err, data) => {
-    if (err) {
-      console.log(err);
-      res.status(400).send();
-    } else {
-      res.status(200).send(data);
-    }
-  });
-});
-
-app.get('/popularDishes/getItems', (req, res) => {
-  const { restaurantId } = req.query;
-  model.getDishes(restaurantId, (err, data) => {
-    if (err) {
-      console.log(err);
-      res.status(400).send();
-    } else {
-      res.status(200).send(data);
-    }
-  });
-});
-
-app.get('/popularDishes/getPhotos', (req, res) => {
-  const { dishId } = req.query;
-  model.getPhotos(dishId, (err, data) => {
-    if (err) {
-      console.log(err);
-      res.status(400).send();
-    } else {
-      res.status(200).send(data);
-    }
-  });
-});
-
-app.get('/popularDishes/getReviews', (req, res) => {
-  const { numberOfReviews } = req.query;
-  model.getReviews(numberOfReviews, (err, data) => {
-    if (err) {
-      console.log(err);
-      res.status(400).send();
-    } else {
-      res.status(200).send(data);
-    }
-  });
-});
-
-app.get('/popularDishes/arrow.png', (req, res) => {
-  res.sendFile('/arrow.png');
-});
-
-app.get('/popularDishes/leftarrow.png', (req, res) => {
-  res.sendFile('/leftarrow.png');
-});
+require('dotenv').config();
+const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
+const Controller = require('./controllers/mongo.controller.js');
+
+const app = express();
+
+app.listen(process.env.SERVER_PORT, (err) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log(`Server is running on ${process.env.SERVER_PORT}`);
+  }
+});
+
+app.use(bodyParser.json());
+app.use('/', express.static(path.join(__dirname, '../client/dist/')));
+
+/*
+  * POST ROUTES
+*/
+
+app.post('/api/dishes', (req, res) => {
+  Controller.postDish(req, res);
+});
+
+app.post('/api/reviews', (req, res) => {
+  Controller.postReview(req, res);
+});
+
+/*
+  * GET ROUTES
+*/
+
+app.get('/api/restaurants/:restaurantId', (req, res) => {
+  Controller.getDishes(req.params.restaurantId, res);
+});
+
+app.get('/api/restaurants/:restaurantId/nearby', (req, res) => {
+  Controller.getNearby(req.params.restaurantId, res);
+});
+
+app.get('/popularDishes/arrow.png', (req, res) => {
+  res.sendFile(path.join(__dirname, '/assets/arrow.png'));
+});
+
+app.get('/popularDishes/leftarrow.png', (req, res) => {
+  res.sendFile(path.join(__dirname, '/assets/leftarrow.png'));
+});
+
+
+/*
+  * PUT ROUTES
+*/
+
+app.put('/api/dishes/', (req, res) => {
+  Controller.putDish(req, res);
+});
+
+app.put('/api/reviews/', (req, res) => {

```

Comment:
The PUT routes end with '/' but the others don't. Is that a naming convention?

Reply:
Thanks I fixed that and remove the commented route at the bottom.

Reply:
👍